paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -961,6 +961,74 @@ class TransposeMKLDNNHandler : public MKLDNNHandler {
   std::vector<int> logical_axis_;
 };
 
+template <typename T>
```

Comment:
I have a sentiment to this Handler but It is not used in this PR

Reply:
@jczaja Do you think that I should remove this handler since as you said "It is not used in this PR" ?

Reply:
Long term we hope to use it so I would leave it like that , but I'm not sure if coverage CI will allow it to stay not used.